joss/paper.md

```diff
@@ -1,51 +1,120 @@
+    index: 2
+  - name: Department of Physics and Astronomy, University of Kansas, 1082 Malott, 1251 Wescoe Hall Dr., Lawrence, KS 66045, USA
+    index: 3
+  - name: Department of Astronomy and Astrophysics, University of California, Santa Cruz, 1156 High Street, Santa Cruz, CA 95064, USA
+    index: 4
+  - name: Department of Astronomy & Astrophysics, University of Chicago, 5640 S. Ellis Avenue, Chicago, IL 60637, USA
+    index: 5
+  - name: Department of Astronomy, University of Maryland, College Park, MD USA
+    index: 6
+  - name: Steward Observatory, University of Arizona, Tucson, AZ 85719, USA
+    index: 7
+  - name: Max-Planck-Institut für Astronomie, Königstuhl 17, D-69117 Heidelberg, Germany
+    index: 8
+  - name: Department of Physics and Institute for Research on Exoplanets, Université de Montréal, Montreal, QC, Canada
+    index: 9
+  - name: Space Telescope Science Institute, 3700 San Martin Drive, Baltimore, MD 21218, USA
+    index: 10
+  - name: Johns Hopkins APL, 11100 Johns Hopkins Road, Laurel, MD 20723, USA
+    index: 11
+date: 30 May 2022
 bibliography: paper.bib
 
 ---
 
 # Summary
 
-Eureka! is a data reduction and analysis pipeline for exoplanet time series observations, with a particular focus on James Webb observations. The pipeline is end-to-end, and includes spectral extraction, light curve fitting, high level outputs, ... 
+`Eureka!` is a data reduction and analysis pipeline for exoplanet time-series observations, with a particular focus on James Webb Space Telescope (JWST) observations. The goal of `Eureka!` is to provide an end-to-end pipeline which starts with raw, uncalibrated FITS files and ultimately results in high quality exoplanet transmission and/or emission spectra. The pipeline has a modular structure with six stages, and each stage uses a "Eureka! Control File" (ECFs use the .ecf file extension) to allow for easy control of the pipeline's behavior. We have provided template ECFs for the MIRI, NIRCam, NIRISS, and NIRSpec instruments on JWST and the WFC3 instrument on the Hubble Space Telescope (HST). These templates give users a good starting point for their analyses, but `Eureka!` is not intended to be used as a black-box tool, and users should expect to fine-tune some settings for each observation in order to achieve optimal results. Throughout the pipeline, many intermediate figures and outputs are created to allow users to compare `Eureka!`'s performance using different parameter settings or to compare `Eureka!` with an independent pipeline. The ECF used to run each stage is also copied into the output folder from each stage to increase reproducibility. Finally, while `Eureka!` has been optimized for exoplanet observations (especially the later stages of the code), much of the core functionality could also be repurposed for JWST time-series observations in other research domains thanks to `Eureka!`'s modularity.
```

Comment:
"uses a "Eureka! Control File" (ECFs use the .ecf file extension)" doesn't read quite right to me and the acronym isn't quite formally introduced. Maybe: "uses a "Eureka! Control File" (ECF; these all use the .ecf file extension)"